### PR TITLE
Add content about 3.6.1 in changelog.

### PR DESCRIPTION
**Pull request by chenrujun**
Add content about 3.6.1 in changelog.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to view this PR.

File: sdk/spring/azure-spring-boot/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## 3.7.0-beta.1 (Unreleased)
 
+## 3.6.1 (2021-07-02)
+### New Features
+- Upgrade to [spring-boot-dependencies:2.5.2](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.2/spring-boot-dependencies-2.5.2.pom).
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot/CHANGELOG.md:7]: We need to add changelog to azure-spring-boot-starter as well.

**Review by yiliuTo — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot/CHANGELOG.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot/CHANGELOG.md:7]: Already added.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**